contracts/0.8.25/lib/SSZ.sol

```diff
@@ -19,6 +19,17 @@ library SSZ {
     error InvalidPubkeyLength();
     error InvalidBlockHeader();
 
+    /// @notice Domain for deposit message signing
+    /// @dev per https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#domain-types
+    bytes4 internal constant DOMAIN_DEPOSIT = 0x03000000;
+
+    /// @notice calculation of deposit domain based on fork version
+    /// @dev per https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#compute_domain
+    function computeDepositDomain(bytes4 genesisForkVersion) internal view returns (bytes32 depositDomain) {
+        bytes32 forkDataRoot = sha256Pair(genesisForkVersion, bytes32(0));
+        depositDomain = DOMAIN_DEPOSIT | (forkDataRoot >> 32);
```

Comment:
```
        depositDomain = DOMAIN_DEPOSIT | (forkDataRoot >> 32);
```
Is `|` really equal to `+` in the spec here? Not really proficient in python, but doubting.